ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -755,19 +755,22 @@ objects:
           fi
         fi
 
-        if has_variant "ipv6"; then
-          export OPENSHIFT_INSTALL_AZURE_EMULATE_SINGLESTACK_IPV6=true
+        if has_variant "dualstack"; then
+          export OPENSHIFT_INSTALL_EXPERIMENTAL_DUAL_STACK=true
           cat >> ${ARTIFACT_DIR}/installer/install-config.yaml << EOF
         networking:
           networkType: OVNKubernetes
           machineNetwork:
-            - cidr: 10.0.0.0/16
-            - cidr: fd00::/48
+          - cidr: 10.0.0.0/16
+          - cidr: fd00::/48
```

Comment:
(I feel like the form with indentation is preferred...)

Reply:
I ask myself that question every time, this is my editor linter ... but I see it both ways

Reply:
@danwinship I need lgtm again, I've added the indentantion